diff --git a/node_modules/@agoric/telemetry/src/context-aware-slog.js b/node_modules/@agoric/telemetry/src/context-aware-slog.js
index 1263928..59a9afc 100644
--- a/node_modules/@agoric/telemetry/src/context-aware-slog.js
+++ b/node_modules/@agoric/telemetry/src/context-aware-slog.js
@@ -51,1 +51,1 @@ 
-const SLOG_TYPES = {
+export const SLOG_TYPES = {
@@ -62,2 +62,2 @@     COMMIT: {
-      FINISH: 'cosmic-swingset-commit-finish',
-      START: 'cosmic-swingset-commit-start',
+      FINISH: 'cosmic-swingset-commit-block-finish',
+      START: 'cosmic-swingset-commit-block-start',
@@ -70,0 +70,1 @@     },
+    KERNEL_UPGRADE_EVENTS: 'cosmic-swingset-inject-kernel-upgrade-events',
@@ -75,0 +75,4 @@     },
+    UPGRADE: {
+      FINISH: 'cosmic-swingset-upgrade-finish',
+      START: 'cosmic-swingset-upgrade-start',
+    },
@@ -86,0 +86,1 @@   },
+  CREATE_VAT: 'create-vat',
@@ -232,1 +232,4 @@         if (!finalBody.runNum) {
-          assert(!triggerContext);
+          assert(
+            !triggerContext ||
+              triggerContext['run.trigger.type'] === 'bootstrap',
+          );
